src/components/FormatE1/MagazineSection.tsx

```diff
@@ -11,8 +11,8 @@ export const MagazineSection = () => (
     <div className="sm:hidden">
       <Image src="/images/format-e1/formaty.png" width={400} height={400} alt="czasopismo" className="w-full" />
       <Button kind="primary" className="mx-auto mt-8 flex items-center">
-        Zobacz wszystkie
-        <Image src={WhiteArrowIcon} width={16} height={16} alt="arrow" className="ml-4" />
+        <h3>Zobacz wszystkie</h3>
```

Comment:
It is the same as in the Figma project. Imo customizing paragraphs is just a waste of time because we have the same results just by placing the header tag as in Figma.